src/openms/include/OpenMS/TRANSFORMATIONS/FEATUREFINDER/MultiplexDeltaMassesGenerator.h

```diff
@@ -74,19 +77,55 @@ namespace OpenMS
     void generateKnockoutDeltaMasses();
 
     /**
-     * @brief write the list of labels for each sample
+     * @brief write the list of labels for each of the sample
      */
-    void printLabelsList() const;
+    void printSamplesLabelsList() const;
```

Comment:
would be better to write to a stream so the user can control if he uses a DEBUG or LOG stream.